HumanSpaceProgram/Assets/HSP.Vanilla/ReferenceFrames/FixedReferenceFrameTransform.cs

```diff
@@ -0,0 +1,321 @@
+    {
+        Vector3Dbl _absolutePosition = Vector3Dbl.zero;
+        QuaternionDbl _absoluteRotation = QuaternionDbl.identity;
+
+        public Vector3 Position
+        {
+            get
+            {
+                RecalculateCacheIfNeeded();
+                return _rb.position;
+            }
+            set
+            {
+                this._absolutePosition = SceneReferenceFrameManager.ReferenceFrame.TransformPosition( value );
+                MakeCacheInvalid();
+                ReferenceFrameTransformUtils.SetScenePositionFromAbsolute( transform, _rb, _absolutePosition );
+                OnAbsolutePositionChanged?.Invoke();
+                OnAnyValueChanged?.Invoke();
+            }
+        }
+
+        public Vector3Dbl AbsolutePosition
+        {
+            get => _absolutePosition;
+            set
+            {
+                _absolutePosition = value;
+                MakeCacheInvalid();
+                ReferenceFrameTransformUtils.SetScenePositionFromAbsolute( transform, _rb, value );
+                OnAbsolutePositionChanged?.Invoke();
+                OnAnyValueChanged?.Invoke();
+            }
+        }
+
+        public Quaternion Rotation
+        {
+            get
+            {
+                RecalculateCacheIfNeeded();
+                return _rb.rotation;
+            }
+            set
+            {
+                this._absoluteRotation = SceneReferenceFrameManager.ReferenceFrame.TransformRotation( value );
+                MakeCacheInvalid();
+                ReferenceFrameTransformUtils.SetSceneRotationFromAbsolute( transform, _rb, _absoluteRotation );
+                OnAbsoluteRotationChanged?.Invoke();
+                OnAnyValueChanged?.Invoke();
+            }
+        }
+
+        public QuaternionDbl AbsoluteRotation
+        {
+            get => _absoluteRotation;
+            set
+            {
+                _absoluteRotation = value;
+                MakeCacheInvalid();
+                ReferenceFrameTransformUtils.SetSceneRotationFromAbsolute( transform, _rb, value );
+                OnAbsoluteRotationChanged?.Invoke();
+                OnAnyValueChanged?.Invoke();
+            }
+        }
+
+
+        public Vector3 Velocity
+        {
+            get
+            {
+                RecalculateCacheIfNeeded();
+                return _cachedVelocity;
+            }
+            set { }
+        }
+
+        public Vector3Dbl AbsoluteVelocity
+        {
+            get => Vector3Dbl.zero;
+            set { }
+        }
+
+        public Vector3 AngularVelocity
+        {
+            get
+            {
+                RecalculateCacheIfNeeded();
+                return _cachedAngularVelocity;
+            }
+            set { }
+        }
+
+        public Vector3Dbl AbsoluteAngularVelocity
+        {
+            get => Vector3Dbl.zero;
+            set { }
+        }
```

Comment:
Are those empty setters intentional?

Reply:
Yes, I clarified it in the latest commit.

'Fixed' transform is always "fixed" in place (in absolute space), so it makes no sense to set its motion values to anything.